WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -494,7 +494,22 @@ class PostListViewController: AbstractPostListViewController, UIViewControllerRe
     }
 
     func cell(_ cell: UITableViewCell, handleTrashPost post: AbstractPost) {
-        deletePost(post)
+        if (post.status == PostStatusTrash) {
+
+            let cancelText = NSLocalizedString("Cancel", comment: "Cancels an Action")
+            let deleteText = NSLocalizedString("Delete", comment: "Deletes post permanently")
+            let messageText = NSLocalizedString("Delete Trashed Post", comment: "Deletes post permanently")
```

Comment:
Sorry, one last amend – could you change the key here from "Delete Trashed Post" to "Delete this post permanently?" so it matches the actual text that's displayed? Thank you!